vector/src/main/java/im/vector/app/features/home/room/list/home/HomeRoomListViewModel.kt

```diff
@@ -250,6 +270,38 @@ class HomeRoomListViewModel @AssistedInject constructor(
         }
     }
 
+    private fun getEmptyStateData(filter: HomeRoomFilter, selectedSpace: RoomSummary?): StateView.State.Empty? {
+        return when (filter) {
+            HomeRoomFilter.ALL ->
+                if (selectedSpace != null) {
+                    StateView.State.Empty(
+                            title = stringProvider.getString(R.string.home_empty_space_no_rooms_title, selectedSpace.displayName),
+                            message = stringProvider.getString(R.string.home_empty_space_no_rooms_message),
+                            image = drawableProvider.getDrawable(R.drawable.ill_empty_space),
```

Comment:
OOI what is the meaning of `ill`?

Reply:
illustration

usually icons are vector and big images (illustrations) are pngs or wepb, so it's useful to indicate it in the name

Reply:
OK. `ic_` is used by mistake on other files, we may want to do some renaming later.